linalg/solvers.hpp

```diff
@@ -241,11 +241,13 @@ class IterativeSolver : public Solver
    virtual void SetPrintLevel(PrintLevel);
    ///@}
 
-   /// @name Solver statistics
+   /// @name Solver statistics. These are valid after the call to Mult().
    ///@{
    int GetNumIterations() const { return final_iter; }
    bool GetConverged() const { return converged; }
+   double GetInitialNorm() const { return initial_norm; }
    double GetFinalNorm() const { return final_norm; }
+   double GetFinalRelNorm() const { return final_norm / initial_norm; }
```

Comment:
Maybe we should add Doxygen comments for (some) of these individual member functions?

Reply:
I'm not sure what to write. Feel free to add.

Reply:
I added some comments in 7aa756beb, feel free to edit.